src/EventListener/AdminMenuListener.php

```diff
@@ -21,7 +21,7 @@ public function addAdminMenuItem(MenuBuilderEvent $event): void
     {
         $menu = $event->getMenu();
         if (null !== $content = $menu->getChild('configuration')) {
-            $content->addChild('app-menu', ['route' => 'monsieurbiz_menu_admin_menu_index'])
+            $content->addChild('menu_type', ['route' => 'monsieurbiz_menu_admin_menu_index'])
```

Comment:
Why do we need to update the name here? Something changed in Sylius 2.0 regarding the admin menu builder?